scripts/contentscript.js

```diff
@@ -16,64 +16,133 @@
 // You should have received a copy of the GNU General Public License
 // along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
 
-/*
- *same with $(document).ready(function(){ });
- *Got it from http://youmightnotneedjquery.com/ 
- */
-function ready(fn) {
```

Comment:
@greenlikeorange We don't need this? Could you please explain me so that I could decide.